### PR TITLE
Project 2: Mobile version

### PR DESCRIPTION
**Pull request by binhussen**
# Project 2: Mobile version
In this PR I added :

- Added Works section.
- Added About myself section.
- Used ul and li for categories in the Work and About myself section.
- Used CSS Grid in the main container that contains all of the cards and Flexbox to lay out an individual card.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,6 +27,18 @@
...
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@500&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
```
Comment (wuletawwonte) [inline index.html:44]: - [ ] [OPTIONAL] Please move the logo of the website out of the nav tag.

File: index.html [hunk 2]
```diff
@@ -27,6 +27,18 @@
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
       rel="stylesheet"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (wuletawwonte) [inline index.html:30]: - [ ] [OPTIONAL] Please add license file in the root directory of your project. You can use [Microverses template](https://github.com/microverseinc/readme-template/blob/master/MIT.md).

**Review by wuletawwonte — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @binhussen  👋🏿👋🏿👋🏿 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions

- [ ] [OPTIONAL] Please don't open a new pull request for a re-review. Implement the requested changes by the previous reviewer and use the link to the same pull request to ask for a re-review. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (binhussen) [reply index.html:30]: Thanks, @wuletawwonte for your feedback, I Will Modify It.

File: index.html (hunk 1 above)
Comment (binhussen) [reply index.html:44]: Thanks @wuletawwonte for your feedback, I Will Modify It.

**Comment by binhussen** [comment]
Thank your @wuletawwonte for your feedback and approval.

**Pull request merged**